src/providers/postgres/qgspostgresimportprojectdialog.cpp

```diff
@@ -0,0 +1,254 @@
+/***************************************************************************
+    qgspostgresimportprojectdialog.cpp
+    ---------------------
+    begin                : September 2025
+    copyright            : (C) 2025 by Jan Caha
+    email                : jan.caha at outlook dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include <QHBoxLayout>
+#include <QVBoxLayout>
+#include <QLabel>
+#include <QDir>
+#include <QDirIterator>
+#include <QMenu>
+#include <QFileDialog>
+#include <QHeaderView>
+
+#include "qgspostgresimportprojectdialog.h"
+#include "qgsproject.h"
+#include "qgsguiutils.h"
+#include "qgsapplication.h"
+#include "qgspostgresconn.h"
+#include "moc_qgspostgresimportprojectdialog.cpp"
+
+QgsPostgresImportProjectDialog::QgsPostgresImportProjectDialog( std::shared_ptr<QgsAbstractDatabaseProviderConnection> dbConn, const QString targetSchema, QWidget *parent )
+  : QDialog { parent }, mDbConnection( dbConn ), mSchemaToImportTo( targetSchema )
+{
+  setWindowTitle( tr( "Import Projects to Schema “%1”" ).arg( targetSchema ) );
+  setMinimumWidth( 600 );
```

Comment:
Sorry, this comment somehow did not make it into the review:
Is this not a bit too wide?
Do we need an explicit minimum width?